crates/api_models/src/payments.rs

```diff
@@ -1042,6 +1042,7 @@ pub struct ConnectorMandateReferenceId {
     pub connector_mandate_id: Option<String>,
     pub payment_method_id: Option<String>,
     pub update_history: Option<Vec<UpdateHistory>>,
+    pub mandate_metadata: Option<pii::SecretSerdeValue>,
```

Comment:
keep it as a `serde_json::Value` for now

Reply:
Done